### PR TITLE
Add a Site model (and fix CHANGELOG)

### PR DESCRIPTION
**Pull request by blaix**
I need a site model for some upcoming bottomline changes.

This also contains some CHANGELOG fixes.

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: yolapy/models/site.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            setattr(self, key, val)
+
+    @classmethod
+    def get(cls, site_id):
+        """Get a site from the Yola API."""
+        site_attributes = Yola().get_site(site_id)
+        return Site(**site_attributes)
+
+    @property
+    def published_domain(self):
```
Comment (RayeN) [inline yolapy/models/site.py]: This will work only for Yola publishing.

**Comment by RayeN** [comment]
:+1: Looks good to me.

File: yolapy/models/site.py (hunk 1 above)
Comment (blaix) [inline yolapy/models/site.py]: how do we get a domain for partner sites? do we need sbbe to expose `partner_domain`?

Comment (RayeN) [inline yolapy/models/site.py]: I think it's already exposed as Site's attribute.

Comment (RayeN) [inline yolapy/models/site.py]: Nope it isn't... It's only part of site content atm, not site's meta :(

Comment (blaix) [inline yolapy/models/site.py]: As we discovered together in slack, it's only exposed on the site content resource, which isn't available in the Yola API. I'm going to add it to site meta and then update this pull.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by blaix** [comment]
@RayeN I updated the model so we will have to use partner_domain, not publishing data, as we discussed.

**Comment by RayeN** [comment]
:+1:

**Pull request merged**